web/src/app/main/components/ToolbarTitle.js

```diff
@@ -32,6 +32,10 @@ const styles = {
   title: {
     padding: '0 4px 0 4px',
     flex: 1, // pushes toolbar actions to the right
+    fontWeight: 500,
```

Comment:
it would make more sense to add this to a separate PR, especially one where we can tackle some more accessibility issues all at once.